spec/services/content_renderer_spec.rb

```diff
@@ -1,93 +1,88 @@
 require "rails_helper"
 
 RSpec.describe ContentRenderer do
-  describe "#process" do
-    let(:markdown) { "hello, hey" }
-    let(:expected_result) { "<p>hello, hey</p>\n\n" }
-    let(:mock_fixer) { class_double MarkdownProcessor::Fixer::FixAll }
-    let(:mock_front_matter_parser) { instance_double FrontMatterParser::Parser }
-    let(:mock_processor) { class_double MarkdownProcessor::Parser }
-    let(:fixed_markdown) { :fixed_markdown }
-    let(:parsed_contents) { Struct.new(:content, :front_matter).new(:parsed_content) }
-    let(:processed_contents) { instance_double MarkdownProcessor::Parser }
+  let(:markdown) { "hello, hey" }
+  let(:renderer) { described_class.new(markdown, source: nil, user: nil, fixer: MarkdownProcessor::Fixer::FixAll) }
 
-    # rubocop:disable RSpec/InstanceVariable
-    before do
-      allow(mock_fixer).to receive(:call).and_return(fixed_markdown)
-      allow(mock_front_matter_parser).to receive(:call).with(fixed_markdown).and_return(parsed_contents)
-      allow(mock_processor).to receive(:new).and_return(processed_contents)
-      allow(processed_contents).to receive(:finalize).and_return(expected_result)
-      @original_fixer = described_class.fixer
-      @original_parser = described_class.front_matter_parser
-      @original_processor = described_class.processor
+  it "calls fixer" do
+    allow(MarkdownProcessor::Fixer::FixAll).to receive(:call).and_call_original
+    described_class.new(markdown, source: nil, user: nil, fixer: MarkdownProcessor::Fixer::FixAll).process
+    expect(MarkdownProcessor::Fixer::FixAll).to have_received(:call).with(markdown)
+  end
 
-      described_class.fixer = mock_fixer
-      described_class.front_matter_parser = mock_front_matter_parser
-      described_class.processor = mock_processor
+  describe "#process" do
+    let(:parser) { instance_double(MarkdownProcessor::Parser) }
+
+    context "with double parser" do
```

Comment:
"double" parser here is a reference to `instance_double`, not having :two:x parsers?